packages/openai/src/responses/openai-responses-language-model.test.ts

```diff
@@ -63,68 +63,72 @@ describe('OpenAIResponsesLanguageModel', () => {
-            instructions: null,
-            max_output_tokens: null,
-            model: 'gpt-4o-2024-07-18',
-            output: [
-              {
-                id: 'msg_67c97c02656c81908e080dfdf4a03cd1',
-                type: 'message',
-                status: 'completed',
-                role: 'assistant',
-                content: [
-                  {
-                    type: 'output_text',
-                    text: 'answer text',
-                    annotations: [],
-                  },
-                ],
-              },
-            ],
-            parallel_tool_calls: true,
-            previous_response_id: null,
-            reasoning: {
-              effort: null,
-              summary: null,
+        prepareJsonResponse({
+          id: 'resp_67c97c0203188190a025beb4a75242bc',
+          object: 'response',
+          created_at: 1741257730,
+          status: 'completed',
+          error: null,
+          incomplete_details: null,
+          input: [],
+          instructions: null,
+          max_output_tokens: null,
+          model: 'gpt-4o-2024-07-18',
+          output: [
+            {
+              id: 'msg_67c97c02656c81908e080dfdf4a03cd1',
+              type: 'message',
+              status: 'completed',
+              role: 'assistant',
+              content: [
+                {
+                  type: 'output_text',
+                  text: 'answer text',
+                  annotations: [],
+                },
+              ],
             },
-            store: true,
-            temperature: 1,
-            text: {
-              format: {
-                type: 'text',
-              },
+          ],
+          parallel_tool_calls: true,
+          previous_response_id: null,
+          reasoning: {
+            effort: null,
+            summary: null,
+          },
+          store: true,
+          temperature: 1,
+          text: {
+            format: {
+              type: 'text',
             },
-            tool_choice: 'auto',
-            tools: [],
-            top_p: 1,
-            truncation: 'disabled',
-            usage: {
-              input_tokens: 345,
-              input_tokens_details: {
-                cached_tokens: 234,
-              },
-              output_tokens: 538,
-              output_tokens_details: {
-                reasoning_tokens: 123,
-              },
-              total_tokens: 572,
+          },
+          tool_choice: 'auto',
+          tools: [],
+          top_p: 1,
+          truncation: 'disabled',
+          usage: {
+            input_tokens: 345,
+            input_tokens_details: {
+              cached_tokens: 234,
             },
-            user: null,
-            metadata: {},
+            output_tokens: 538,
+            output_tokens_details: {
+              reasoning_tokens: 123,
+            },
+            total_tokens: 572,
           },
-        };
+          user: null,
+          metadata: {},
+        });
```

Comment:
@lgrammel for now Ileft the `prepareJsonResponse()` in the `beforeEach()` block. Do you want me to move into each `it()` block and adjust the mock for each test depending on what's relevant?

Reply:
would be great. most tests follow a pattern where the test setup is prepared in each it statement since they are somewhat different and to get good test coverage i try to have realistic test inputs for each (i.e also no unrelated data in test setup)